lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -32,16 +32,25 @@ const _flatten = arr => [].concat.apply([], arr);
  * @param {!Connection} connection
  * @param {string} url
  * @param {!Object} options Lighthouse options.
- * @param {!Array<string>} requestedAudits Names of audits to run.
+ * @param {!Array<string>} requestedAuditNames Names of audits to run.
  * @return {!Promise}
  */
-window.runLighthouseForConnection = function(connection, url, options, requestedAudits) {
+window.runLighthouseForConnection = function(connection, url, options, requestedAuditNames) {
   // Always start with a freshly parsed default config.
   const runConfig = JSON.parse(JSON.stringify(defaultConfig));
 
   // Filter out audits not requested.
```

Comment:
Can you beef up the comment:
"Create a Map lookup from audit path names to audit names, then filter out audits that shouldn't run by name."